extension/js/scripts.js

```diff
@@ -26,6 +31,9 @@ function package_view(dir) {
 
         var toInstall = [];
         views.select_to_install(toInstall);
+
+        $('#packageview').show();
+        $('#loadingview').hide();
     });
```

Comment:
I see 2 problems:
1. You show if it succeeds, right? And if it fails? Will it continue to show the loading without telling anything to the user and without showing the list/add pkgs view again?
1. And if it fails (i.e network connectivity), will the user have to add new packages to install again, from scratch? (for this problem I don't have a strong opinion. Let's keep it as is)